tiles/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM sourcepole/t-rex
+
+COPY ./config.toml /
+
+ENTRYPOINT /usr/bin/t_rex serve -c /config.toml
```

Comment:
Maybe add `--simplify true`

Reply:
This option is already set in the `config.toml` file.

Reply:
ok my bad !